enterprise/app/esbuild.config.mjs

```diff
@@ -10,7 +10,12 @@ let browserifyPathPlugin = {
     // detect if we are building from buildbuddy-internal repo and
     // adjust 'node_modules' path accordingly
     if (!existsSync(entry)) {
-      entry = path.join(process.cwd(), "external/com_github_buildbuddy_io_buildbuddy", "node_modules");
+      // WORKSPACE
+      entry = path.join(process.cwd(), "external", "com_github_buildbuddy_io_buildbuddy", "node_modules");
+    }
+    if (!existsSync(entry)) {
```

Comment:
did you mean to invert the if condition here? or use `else`?

Reply:
This is meant to be a second fallback path, so I think that neither inverting the condition nor using an `else` would be correct

Reply:
Ah, I can nest the ifs to make this more clear